### ISSUE TITLE
Discussion: FRONT-END technologies

### ISSUE THREAD

**Issue by miggs125** [Seneca-CDOT/telescope#3]
Part of the [MVP features](https://github.com/Seneca-CDOT/telescope/blob/master/docs/overview.md#mvp-features) is to determine what technologies to use. This is a discussion on what technologies to use for the front-end. We should be able to create a poll so everyone can voice their opinion.

**Comment by humphd** [comment]
For an MVP, I'd encourage us to think "static" vs. dynamic web app.  Could we do this with something pre-baked to start?  For example, [11ty](https://www.11ty.io) will let you take a bunch of markdown, html, etc. and produce a static site.

If people want to get into fancy frontends, that's great.  I just don't want to block on that, and don't want to exclude people from making progress early because we get enamoured with a framework war.

**Comment by ODAVING** [comment]
While 11ty is a decent option, I was wondering if we could possibly use Angular6+ to make sure the site can be easily upgraded and become future proof

**Comment by humphd** [comment]
I'm happy for people to experiment with Angular, React, etc. and for more than one front-end to happen in parallel.  We don't even have to have it all happen in this repo (e.g., `telescope-ng` vs. `telescope-react` repos can be made).

However, I'd encourage us to not start with a technology stack, but rather with a mockup of what we want any stack to produce.  Until we have a thing to build, *how* we build it is secondary.

**Comment by ODAVING** [comment]
@humphd do you think it’d be possible for me to tackle the html static page? Or should we split it up into further components?

**Comment by Reza-Rajabi** [comment]
Although it seems we have decided to start with a static web page as a warm-up, I think because of the number of developers, we might better think of a component base frontend such as React, Angular, or Vue. Anyhow, assuming we might start such a component design in the future or parallel, I will make a new issue concerning about UI design and Mockups.

**Comment by humphd** [comment]
We've got an initial MVP front end, so let's close this.  We'll iterate on it more, probably rewriting in React or something in the next iteration.
